setup.py

```diff
@@ -18,41 +18,9 @@ def read(fname):
 exec(read(path.join(here, 'petabtests', 'version.py')))
 
 setup(
-    name='petabtests',
     version=__version__,  # noqa: F821
```

Comment:
Can move this too I think
```toml
[tool.setuptools.dynamic]
version = {attr = "petab.__version__"}
```

Reply:
Yes, it's possible, but this will require importing the package, which will require all module-level import dependencies to be installed in the temporary environment for building the sdist. Not sure it's worth it.